wgsl/index.bs

```diff
@@ -3393,6 +3393,18 @@ short_circuit_or_expression
 
 # Statements TODO # {#statements}
 
+## Compound Statement ## {#compound-statement}
```

Comment:
any reason to not call it a "Block" like in SPIR-V?

Reply:
It's not even the same the block you're thinking of.  I think you're thinking of "basic block" which is a different thing:  a basic block is a sequence of instructions to be executed without interruption by control flow (aside form a function call and a potential discard).

Also, even in SPIR-V land it can be confused with  "block" as in buffer-block, which WGSL already as in the `block` attribute.
It's better to steer clear of the ambiguity.

Reply:
Makes sense, yeah